openupgradelib/openupgrade.py

```diff
@@ -1935,6 +1937,119 @@ def m2o_to_m2m(cr, model, table, field, source_field):
     return m2o_to_x2m(cr, model, table, field, source_field)
 
 
+def m2m_to_o2m(
+    env,
+    model,
+    field,
+    source_relation_table,
+    relation_source_field,
+    relation_comodel_field,
+):
+    """Transform many2many relations into one2many (with possible data
+    loss).
```

Comment:
Could you explicit dataloss. With some exemple like : 
In VX : a1 related to B1, B2, B3. A2 related to B3, B4 , B5.
In VX+1 : ?

Reply:
added better documentation + example.